doc/release-notes.md

```diff
@@ -10,6 +10,43 @@ Option handling
 - The `-reindex` and `-reindex-chainstate` options now imply `-rescan`
   (provided that the wallet is enabled and pruning is disabled, and unless
   `-rescan=0` is specified explicitly).
+- A new string-valued option, `-allowdeprecated` has been introduced to allow
+  a user to explicitly manage the availability of deprecated zcashd features.
+  This flag makes it possible for users to reenable deprecated methods and
+  features api that are currently disabled by default, or alternately to
+  explicitly disable all deprecated features if they so choose. Multiple
+  instances of this argument may be provided. In the case that no values are
+  provided for this parameter, all currently deprecated RPC methods that are
+  not slated for removal in the next release remain enabled. A user may
+  disable all deprecated features entirely by providing the string "none" as
+  the argument to this parameter. In the case that "none" is specified,
+  multiple invocations of `-allowdeprecated` are not permitted.
+
+  Specific user-oriented documentation of the deprecation process may be 
+  found in `doc/book/src/user/deprecation.md`.
```

Comment:
This will end up in the GitHub release notes and blog post, so linkify it:
```suggestion
  found at https://zcash.github.io/zcash/user/deprecation.html.
```